hudi-utilities/src/test/resources/delta-streamer-config/jdbc-source-password.txt

```diff
@@ -0,0 +1 @@
+hudi
```

Comment:
please make sure this is not any real password :)

Reply:
I think we can get rid of this file. it is not even been used in the Junits

Reply:
:) we dont have such strong passwords .. haha